authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.7 on 2021-03-31 18:50
+
+from django.db import migrations, models
+import django.db.models.deletion
```

Comment:
Permissions must be loaded into the DB before group migration can occur.  The group migration file "as-is" lists a dependency on 0001_initial, but this 0001_initial is missing all of the permission models.

I personally don't think it makes sense to track migration files in the repo this way.  With the exception of group_migrations, they are an output generated by Django.  This file also lists a dependency which does not appear to have been pushed ('0012_alter_user_first_name_max_length').  So, the migrations stored in the repo have dependencies which cannot be satisfied.

For the above reasons, I think we should not be pushing migrations files, and group_migrations should be kept in a separate directory (previously was in a post_first_migration folder) since the groups must be migrated after permissions.